test/Microsoft.NET.Build.Tests/GivenThatWeWantToVerifyNuGetReferenceCompat.cs

```diff
@@ -180,6 +141,140 @@ public class GivenThatWeWantToVerifyNuGetReferenceCompat : SdkTest, IClassFixtur
             }
         }
 
+        [WindowsOnlyTheoryAttribute]
+        [InlineData("netstandard2.0")]
+        [InlineData("netcoreapp2.0")]
+        public void Net461_is_implicit_for_Netstandard_and_Netcore_20(string targetFramework)
+        {
+            if (UsingFullFrameworkMSBuild)
```

Comment:
While we're getting better at xunit, should we have some sort of attribute to indicate core msbuild only?